test/src/cases/01_react_logo/Case01_Paper.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import paper, {Group, Path} from "paper";
+import {drawGridLines} from "../common";
+
+export default class Case01_Paper extends React.Component {
+
+  componentDidMount() {
+    paper.install(window);
```

Comment:
@kota65535 testing this library this way makes no sense. I just recently reviewed tests a little bit and was surprised, when I saw you used `paper` module. Anyway, it does not matter, I have created a new test [example](https://github.com/react-paper/react-paper-bindings/blob/master/test/index.test.js) using `jest`, `enzyme` and `canvas`. No more headless chrome or selenium stuff, just pure node ;)

Also released version 2.0.0 :)

It would be cool though, if some if not all the test cases are ported to the "new way". But I'm pretty busy ATM.